src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/libra/LibraCourtCaseListenerIntTest.kt

```diff
@@ -99,7 +99,7 @@ class LibraCourtCaseListenerIntTest : MessagingMultiNodeTestBase() {
         "SOURCE_SYSTEM" to "HMCTS",
         "EVENT_TYPE" to LIBRA_COURT_CASE.name,
         "RECORD_COUNT" to "1000000",
-        "SEARCH_VERSION" to "1",
+        "SEARCH_VERSION" to "1.1",
```

Comment:
Maybe this should be a property? It might get annoying if we have to change it a lot as we add tests

Reply:
Moved into a Property